src/main/java/org/jenkinsci/test/acceptance/selenium/Scroller.java

```diff
@@ -83,7 +87,11 @@ public void beforeChangeValueOf(WebElement element, WebDriver driver, CharSequen
         scrollIntoView(element, driver);
     }
 
-    private void scrollIntoView(WebElement e, WebDriver driver) {
+    /**
+     * The framework is expected to take care of the correct scrolling. When you are tempted to scroll from PageObjects
+     * or tests, there is likely a framework problem to be fixed.
```

Comment:
With this fix, by adding the scroll inside the `select` method, we are scrolling generally with all the selects. According to this new comment, it should not be needed. If the framework manage the scroll, shouldn't we leave the fix only for the specific test failing? :thinking:

Reply:
By *framework* I meant the ATH framework. Put differently, we would like the helper classes such as `Control` work without any hack in tests or page objects. Can you think of better formulation for the comment?

Reply:
No, that's right. I got it now, thanks